src/prognostic_equations/remaining_tendency.jl

```diff
@@ -37,6 +37,99 @@ NVTX.@annotate function hyperdiffusion_tendency!(Yₜ, Yₜ_lim, Y, p, t)
     apply_hyperdiffusion_tendency!(Yₜ, Y, p, t)
 end
 
+using ClimaCore.RecursiveApply: rzero
+
+#####
+##### Cell center tendencies
+#####
+function ᶜremaining_tendency(ᶜY, ᶠY, p, t)
```

Comment:
I can't figure out what this function does by its name so a docstring might help